app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -62,13 +75,49 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
     override val learnMoreClicks: Observable<Unit>
         get() = view!!.detailLearnMore.clicks()
 
+    override val kebabMenuClicks: Observable<Unit>
+        get() = view!!.toolbar.kebabMenu.clicks()
+
     override var isPasswordVisible: Boolean = false
         set(value) {
             assertOnUiThread()
             field = value
             updatePasswordVisibility(value)
         }
 
+    private var _menuItemSelection = PublishSubject.create<RouteAction.EditItemMenu>()
+    override val menuItemSelection: Observable<RouteAction.EditItemMenu> = _menuItemSelection
+
+    private val menuOptions: Array<RouteAction.EditItemMenu>
+        get() = RouteAction.EditItemMenu.values()
+
+    private fun setupSpinner(view: View) {
```

Comment:
nit: This should be named for the logical thing it's setting up (e.g., `setupKebabMenu`).  Future us will thank you (-: